mix.lock

```diff
@@ -2,6 +2,8 @@
   "certifi": {:hex, :certifi, "0.3.0"},
   "comeonin": {:hex, :comeonin, "2.0.0"},
   "connection": {:hex, :connection, "1.0.2"},
+  "cors_plug": {:hex, :cors_plug, "0.1.4"},
+  "corsica": {:hex, :corsica, "0.4.0"},
```

Comment:
corsica was the old CORS lib I was using, it's not in our mix file but the lock file still shows it... probably should clean that up, though doesn't effect anything since it's not used.